README.md

```diff
@@ -6,8 +6,20 @@
 * Requires no database.
 * Trivial to modify.
 
-To get started just `pip install -r requirements.txt` and run `./run.sh `. When you are ready to deploy run
-`./freeze.sh` to get a static version of the site. 
+## Get Started
+```bash
+# Create and activate a virtualenv
+virtualenv --python=python3.7 venv
+source venv/bin/activate
+
+# Install Python packages
+pip install -r requirements.txt
+
+# Run the main script
+./run.sh
+````
```

Comment:
changed this to use a virtualenv

Reply:
hmm... it seems that this breaks the `index.html` when executing `./run.py`.
I'm not sure how to fix this.
If there is no easy fix, I'll revert the changes here.

Reply:
I'm going to remove virtualenv, you can use it, but it's certainly not required.